teuthology/__init__.py

```diff
@@ -38,6 +48,8 @@
     format='%(asctime)s.%(msecs)03d %(levelname)s:%(name)s:%(message)s')
 log = logging.getLogger(__name__)
 
+log.info('teuthology version: %s', __version__)
```

Comment:
Well, the other one is only to the log, so not in anyone's face, particularly if they're doing -suite or something.  I was aiming for "in your face" as a goad to get people to verify that they're running the latest.  But I can definitely change it or remove it.  Some ideas, some of which we discussed:

1) at least not for help/usage, but only on a valid command (don't know how hard this is; might involve touching every entry point).  Although this does give us a version-check-only for free.
2) only on --version/-V?
3) .debug instead of .info
4) ???
5) Profit

I'll bow to your recommendation; I don't feel that strongly and we can change minds easily later.